packages/fxa-components/Survey/index.tsx

```diff
@@ -5,7 +5,7 @@
 import React, { useState } from 'react';
 import { CSSTransition } from 'react-transition-group';
 
-import './index.scss';
+// import './index.scss';
```

Comment:
Delete it if we don't need it.

Reply:
We want to have styles declared next to the components. However the css ADR isn't complete, I've spoken with @jodyheavener about this, and the plan is to have it ironed out during the settings work. Then we can loop back around and remove the survey.scss file in content server and use the single stylesheet here